modules/aolBidAdapter.js

```diff
@@ -1,341 +1,346 @@
+}
+
+function _isNexageRequestGet(bid) {
+  return _isNexageBidder(bid.bidder) && bid.params.dcn && bid.params.pos;
+}
+
+function isMarketplaceBid(bid) {
+  return _isMarketplaceBidder(bid.bidder) && bid.params.placement && bid.params.network;
+}
+
+function isMobileBid(bid) {
+  return _isNexageRequestGet(bid) || _isNexageRequestPost(bid);
+}
+
+function resolveEndpointCode(bid) {
+  if (_isNexageRequestGet(bid)) {
+    return AOL_ENDPOINTS.MOBILE.GET;
+  } else if (_isNexageRequestPost(bid)) {
+    return AOL_ENDPOINTS.MOBILE.POST;
+  } else if (isMarketplaceBid(bid)) {
+    return AOL_ENDPOINTS.DISPLAY.GET;
   }
+}
 
-  function _isMarketplaceRequest(bid) {
-    return _isMarketplaceBidder(bid.bidder) && bid.params.placement && bid.params.network;
-  }
+function formatBidRequest(endpointCode, bid) {
+  let bidRequest;
+
+  switch (endpointCode) {
+    case AOL_ENDPOINTS.DISPLAY.GET:
+      bidRequest = {
+        url: _buildMarketplaceUrl(bid),
+        method: 'GET'
+      };
+      break;
 
-  function _callBids(params) {
-    utils._each(params.bids, bid => {
-      let apiUrl;
-      let data = null;
-      let options = {
-        withCredentials: true
+    case AOL_ENDPOINTS.MOBILE.GET:
+      bidRequest = {
+        url: _buildOneMobileGetUrl(bid),
+        method: 'GET'
       };
-      let isNexageRequestPost = _isNexageRequestPost(bid);
-      let isNexageRequestGet = _isNexageRequestGet(bid);
-      let isMarketplaceRequest = _isMarketplaceRequest(bid);
-
-      if (isNexageRequestGet || isNexageRequestPost) {
-        apiUrl = _buildNexageApiUrl(bid);
-        if (isNexageRequestPost) {
-          data = bid.params;
-          options.customHeaders = {
+      break;
+
+    case AOL_ENDPOINTS.MOBILE.POST:
+      bidRequest = {
+        url: _buildOneMobileBaseUrl(bid),
+        method: 'POST',
+        data: bid.params,
+        options: {
+          contentType: 'application/json',
+          customHeaders: {
             'x-openrtb-version': '2.2'
-          };
-          options.method = 'POST';
-          options.contentType = 'application/json';
+          }
         }
-      } else if (isMarketplaceRequest) {
-        apiUrl = _buildMarketplaceUrl(bid);
-      }
+      };
+      break;
+  }
 
-      if (apiUrl) {
-        ajax(apiUrl, response => {
-          // Needs to be here in case bidderSettings are defined after requestBids() is called
-          if (showCpmAdjustmentWarning &&
-            $$PREBID_GLOBAL$$.bidderSettings && $$PREBID_GLOBAL$$.bidderSettings.aol &&
-            typeof $$PREBID_GLOBAL$$.bidderSettings.aol.bidCpmAdjustment === 'function'
-          ) {
-            utils.logWarn(
-              'bidCpmAdjustment is active for the AOL adapter. ' +
-              'As of Prebid 0.14, AOL can bid in net – please contact your accounts team to enable.'
-            );
-          }
-          showCpmAdjustmentWarning = false; // warning is shown at most once
+  bidRequest.bidderCode = bid.bidder;
```

Comment:
Any reason to add these param to `bidRequest`. I don't see its use.

Reply:
Hi. We have an analytic adapter that is not part of Prebid official yet. Having "bidder Code" here allows us to break out the reporting between different aliases.